check/fixes.frm

```diff
@@ -2668,3 +2668,54 @@ L F = #x;
 .end
 assert compile_error?("Illegal position for #")
 *--#] Issue499_2 :
+*--#[ Issue512_1 :
+#-
+* Sort which fills SmallExtension:
+
+#: SmallSize 1000K
+#: SmallExtension 1200K
+#: SubSmallSize 1000K
+#: SubSmallExtension 1200K
+#: SubTermsInSmall 2M
+
+Symbol x,n;
+CFunction f,g,prf;
+
+Local test = (<f(1)>+...+<f(150)>)*(<g(1)>+...+<g(100)>);
+.sort
+
+Off parallel;
+PolyRatFun prf;
+
+Identify f(x?) = prf(n-x,n+x);
+
+.end
+assert runtime_error?("Please increase SmallExtension setup parameter.")
```

Comment:
Can you add the following code snippet for ParFORM (after confirming that ParFORM displayed the correct runtime errors in GitHub Actions)?
```
# Runtime errors may freeze ParFORM.
#pend_if mpi?
```